src/probes/journal.c

```diff
@@ -186,6 +186,33 @@ static bool get_boot_id(char **data)
         return true;
 }
 
+#define JOURNAL_MATCH(data) \
+        do { \
+                r = sd_journal_add_match(journal, data, 0); \
+                if (r < 0) { \
+                        tm_journal_match_err(r); \
+                        return false; \
```

Comment:
Always a little scary when the macro can return in some cases but in this case it seems reasonable enough.